### ISSUE TITLE
PhotonVision not Working on Romi Robot

### ISSUE THREAD

**Issue by zagdrath** [PhotonVision/photonvision#274]
I followed both [https://docs.photonvision.org/en/latest/docs/getting-started/installation/romi.html] and [https://docs.photonvision.org/en/latest/docs/getting-started/installation/coprocessor-image.html#other-debian-based-co-processor-installation] while installing PhotonVision to my romi robot and  when I do systemctl status  photonvision.service an error come sup and says this.
![](https://cdn.discordapp.com/attachments/528555967827148801/869342251543855234/unknown.png)

I am unable to access the PhotonVision web interface by going to http://10.0.0.2:5800/ which is what the guide says to go to.

**Comment by zagdrath** [comment]
I do believe the issue is that the systemd service is not running as root even though it should by default.

**Comment by mcm001** [comment]
Yeah that seems to line up. Photonvision needs to run as root on Linux usually yes.

**Comment by Daltz333** [comment]
The solution here is that the system needs to be in RW mode.

```
systemctl stop photonvision.service
rw
systemctl start photonvision.service
```

The install script should detect if the host system is a romi (or have in docs) and disable RW (or other workaround) for the romi since photonvision handles this.

**Comment by Daltz333** [comment]
More information: Romi system can be detected via existence of
```
pi@photonvision(rw):~$ ls /service/wpilibws-romi
log  run  supervise
```

**Comment by gerth2** [comment]
Also update.... the latest development version (and eventual 1.9 release) should have a change which allows running without root. You won't be able to save settings, but it should at least execute.

**Comment by mcm001** [comment]
Closing in favor of #1452
